Dalamud/Interface/Internal/Windows/ChangelogWindow.cs

```diff
@@ -384,4 +517,10 @@ bool CenteredIconButton(FontAwesomeIcon icon, string text)
     public void Dispose()
     {
     }
+
+    private static class FtueLevels
+    {
+        public const int Default = 1;
+        public const int AutoUpdateBehavior = 2;
+    }
```

Comment:
Not _particularly_ a fan of doing things this way - we should instead track whether the user has seen any given screen (at a specific version). Unfortunately I'm also lazy.